app/app-config-schema.json

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
All files in `app/` folder are configurable by the user. This is something internal to fawkes. Pls move this to `app_config/` folder

Reply:
Done